src/components/GradeLevelFilter.jsx

```diff
@@ -2,77 +2,47 @@ import React, { useState } from 'react';
 import { makeStyles } from '@material-ui/core/styles';
 import { Button, Popover } from '@material-ui/core';
 import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+import Autocomplete from '@material-ui/lab/Autocomplete';
+import TextField from '@material-ui/core/TextField';
 
 const useStyles = makeStyles((theme) => ({
-  buttonStyle: {
-    margin: theme.spacing(1),
-    height: theme.spacing(4),
-  },
   popOverWindow: {
-    minWidth: '200px',
-    fontSize: '15px',
-    paddingTop: theme.spacing(1),
-    paddingLeft: theme.spacing(1),
-    paddingBottom: theme.spacing(1),
+    margin: theme.spacing(1),
+    minWidth: '250px',
+    maxWidth: '500px',
   },
 }));
 
-const gradeItems = [
-  '12th - High School',
-  'College Freshman',
-  'College Sophomore',
-  'College Junior',
-  'College Senior',
-  '5th Year',
-  'Post Grad',
+const gradeOptions = [
+  { title: '12th - High School' },
+  { title: 'College Freshman' },
+  { title: 'College Sophomore' },
+  { title: 'College Junior' },
+  { title: 'College Senior' },
+  { title: '5th Year' },
+  { title: 'Post Grad' },
 ];
 
 export default function GradeLevelFilter() {
   const classes = useStyles();
-  const [anchorEl, setAnchorEl] = useState(null);
-  const handleClick = (event) => setAnchorEl(event.currentTarget);
-  const [selectedGrades, setSelectedGrades] = useState(new Set());
-
-  function updateGradeSelection(gradeSelected) {
-    const newSet = new Set(selectedGrades);
-    if (selectedGrades.has(gradeSelected)) {
-      newSet.delete(gradeSelected);
-      setSelectedGrades(newSet);
-      return;
-    }
-    newSet.add(gradeSelected);
-    setSelectedGrades(newSet);
-  }
 
   return (
     <>
-      <Button
-        variant="outlined"
-        className={classes.buttonStyle}
-        onClick={handleClick}
-        endIcon={<ArrowDropDownIcon color="primary" />}>
-        Grade
-      </Button>
-      <Popover
-        open={Boolean(anchorEl)}
-        anchorEl={anchorEl}
-        onClose={() => setAnchorEl(null)}
-        anchorOrigin={{ vertical: 'bottom', horizontal: 'left' }}
-        transformOrigin={{ vertical: 'top', horizontal: 'left' }}>
-        {Object.keys(gradeItems).map((key) => (
-          <div className={classes.popOverWindow}>
-            <input
-              key={key}
-              type="checkbox"
-              onClick={() => updateGradeSelection(gradeItems[key])}
-              checked={selectedGrades.has(gradeItems[key])}
-              value={gradeItems[key]}
-            />
-            {gradeItems[key]}
-            <br />
-          </div>
-        ))}
-      </Popover>
+      <Autocomplete
+        className={classes.popOverWindow}
+        multiple
+        limitTags={1}
```

Comment:
Hmm, I'm not entirely fond of how the rest of the page shifts down as you select more options but then shifts back up once you click outside of the dropdown. Also the autocomplete is a bit wonky to use on mobile :P.

Reply:
Considering that the options for selecting grade level are limited and easily groupable, can we just leave this as a Popover with Checkboxes? Be sure to address the comments from my previous review about following the [Checkbox](https://material-ui.com/components/checkboxes/) Material UI examples.

Reply:
I think the Autocomplete we were talking about in the Figma design refers to the **major** filter, not the grade filter. The grade filter is just a popover with checkboxes.

Reply:
Yeah ok we decided not to go with the auto complete so I just went back to the previous commit with the right changes.